src/Ampersand/FSpec/MetaModels.hs

```diff
@@ -39,9 +39,7 @@ pCtx2Fspec env c = do
         else 
           case violationsOfInvariants fSpec of
             [] -> pure fSpec
-            h:tl -> Errors $ fmap mkInvariantViolationsError $ h NE.:| tl
-
-
+            h:tl -> (Errors . fmap (mkInvariantViolationsError (applyViolText fSpec))) (h NE.:| tl)
```

Comment:
Kwestie van smaak. Ik vind al die haakjes erg lastig. Liever de `$` in dit geval.